src/backend/distributed/commands/transmit.c

```diff
@@ -359,6 +359,30 @@ IsTransmitStmt(Node *parsetree)
 }
 
 
+/*
+ * TransmitStatementUser extracts the user attribute from a
+ * COPY ... (format 'transmit', user '...') statement.
+ */
+char *
+TransmitStatementUser(CopyStmt *copyStatement)
+{
+	ListCell *optionCell = NULL;
```

Comment:
Shall we add `AssertArg(IsTransmitStmt((Node *) copyStatement))` to ensure that all callers of this function are transmit statements? 

This is mostly for future callers not to expected non-transmist statements to have `user` field?